eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -244,14 +244,14 @@ stages:
     - template: ../jobs/vmr-build.yml
       parameters:
         # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
```

Comment:
As the comment here indicates, the SDK diff pipeline YAML needs to be updated to reflect this name change.

Reply:
Thanks for pointing that out. I've updated the pipeline YAML in this commit: https://github.com/dotnet/installer/pull/17840/commits/2ce1a30799b207f34f4bd567bf371254c0edb8d4